src/gui/UserStatusSelector.qml

```diff
@@ -26,285 +26,273 @@ ColumnLayout {
-                            z: 1
-                            color: Style.backgroundColor
-                            border.color: userStatusMessageTextField.borderColor
-                            border.width: Style.normalBorderWidth
-                        }
-
-                        Rectangle { // We need to cover the blue border of the non-radiused rectangle
-                            anchors.fill: parent
-                            anchors.leftMargin: parent.width / 4
-                            anchors.rightMargin: parent.width / 4
-                            anchors.topMargin: Style.normalBorderWidth
-                            anchors.bottomMargin: Style.normalBorderWidth
-                            z: 2
-                            color: Style.backgroundColor
-                        }
+                    Rectangle { // We need to cover the blue border of the non-radiused rectangle
+                        anchors.fill: parent
+                        anchors.leftMargin: parent.width / 4
+                        anchors.rightMargin: parent.width / 4
+                        anchors.topMargin: Style.normalBorderWidth
+                        anchors.bottomMargin: Style.normalBorderWidth
+                        z: 2
+                        color: Style.backgroundColor
                     }
                 }
             }
+        }
 
-            ColumnLayout {
-                Layout.fillWidth: true
-                spacing: 0
-
-                Repeater {
-                    model: userStatusSelectorModel.predefinedStatuses
-
-                    PredefinedStatusButton {
-                        id: control
-                        Layout.fillWidth: true
-                        internalSpacing: Style.standardSpacing + fieldButton.padding + userStatusMessageTextField.padding
+        ScrollView {
+            id: predefeinedStatusesScrollView
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            clip: true
 
-                        emoji: modelData.icon
-                        text: "<b>%1</b> – %2".arg(modelData.message).arg(userStatusSelectorModel.clearAtReadable(modelData))
-                        onClicked: userStatusSelectorModel.setPredefinedStatus(modelData)
-                    }
+            ListView {
+                spacing: 0
+                model: userStatusSelectorModel.predefinedStatuses
+                delegate: PredefinedStatusButton {
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    internalSpacing: Style.standardSpacing + fieldButton.padding + userStatusMessageTextField.padding
+
+                    emoji: modelData.icon
+                    text: "<b>%1</b> – %2".arg(modelData.message).arg(userStatusSelectorModel.clearAtReadable(modelData))
+                    onClicked: userStatusSelectorModel.setPredefinedStatus(modelData)
                 }
             }
```

Comment:
@camilasan @allexzander @mgallien the diffs are big but 90% of them are whitespace changes. I removed a `Column` and moved all of the children into the parent `ColumnLayout`. 

The changes to the predefined status list are here